tests/test_buildoptions.py

```diff
@@ -0,0 +1,118 @@
+import pytest
+from iotedgedev.buildoptions import BuildOptions
+
+pytestmark = pytest.mark.unit
+
+
+def test_filter_build_options():
+    build_options = [
+        "--rm",
+        "-f test",
+        "--file test",
+        "-t image",
+        "--tag image"
+    ]
+    build_options_parser = BuildOptions(build_options)
+    assert not build_options_parser.filter_build_options(build_options)
```

Comment:
See comment in iotedgedev/buildoptions.py line 136, the same information (build_options) is being passed in twice when it could be once